lib/runtime/dart_library.js

```diff
@@ -5,7 +5,7 @@
 /* This file defines the module loader for the dart runtime.
 */
 
-var dart_library;
+var dart_library = typeof module !== "undefined" ? module.exports : undefined;
```

Comment:
can we find a module pattern that works for browser and io.js?
(this here seems like it could be folded into the code below: `dart_library || (dart_library = {})` at the least)